tests/Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```

Comment:
Can you double check the differences between `[dev-packages]` and `[packages]`? I have a feeling that, given that this library is just for testing (which is by definition dev work), these might belong under `[dev-packages]` - but it may make no difference.

Reply:
The `[dev-packages]` contain packages that haven't made it into 'production' yet, basically meaning that you can only install those packages by explicitly specifying the `--dev` argument with pip install. Should I move them? In terms of functionality it won't make a difference.

Reply:
Okay, also good to know, I think this one is okay as-is. Thanks for looking into this.